### ISSUE TITLE
[Broker Service] Create a component which will return the latest schema url for a given concrete entity type (added 13 Oct)

### ISSUE THREAD

**Issue by aaclan-ebi** [ebi-ait/dcp-ingest-central#44]

**Comment by ofanobilbao** [comment]
@aaclan-ebi if this was blocking an issue that was already completed, does that mean this ticket can also be closed?

**Comment by aaclan-ebi** [comment]
Yes, this can be closed. This is already implemented and deployed in prod as part of the broker service.

**Comment by aaclan-ebi** [comment]
Ah sorry, I got confused. I thought this is the endpoint that returns the dereferenced schema. The functionality described in this ticket is not implemented yet. This is a technical task needed for the UI to support adding new metadata.
